modules/git/repo_language_stats_gogit.go

```diff
@@ -45,7 +45,7 @@ func (repo *Repository) GetLanguageStats(commitID string) (map[string]int64, err
 
 	var checker *CheckAttributeReader
 
-	if CheckGitVersionAtLeast("1.7.8") == nil {
```

Comment:
Why is the parenthesis still here?

Reply:
To limit the variable scopes.

ps: this PR is still WIP, too early to be reviewd.

Reply:
What's the purpose of a WIP PR then?

Reply:
For me, to do run the full tests.

And you see, GitHub says:

<img width="863" alt="image" src="https://user-images.githubusercontent.com/2114189/173090314-dcbe5d9d-538d-448f-a699-304eca029950.png">


Only after the button is clicked, then it's ready for review.

Reply:
(ps: I mean `draft` status)

Reply:
Ah, okay. Yeah, that's slightly different from my definition:
For me, a WIP PR is a PR you can already comment on, but where you have to be prepared that it will still (drastically) change.

Reply:
Let wait for more time.

Indeed, the situation of refactoring the git module is more complex than I thought. IMO at the moment it's impossible to achieve the ideal status with a few PRs. We still have to move on.